package.json

```diff
@@ -43,6 +43,7 @@
     "eslint-plugin-react-hooks": "1.0.1",
     "husky": "1.3.1",
     "jest-dom": "3.0.2",
+    "jest-styled-components": "^6.3.1",
```

Comment:
pin the version - remove `^` and install using `yarn add <package> -E`

Reply:
done